example/components/SlidingTabNavigationExample.js

```diff
@@ -17,6 +19,18 @@ export default class SlidingTabNavigationExample extends Component {
     },
   }
 
+  _goToFirstTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('first');
+    });
+  };
+
+  _goToSecondTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('second');
```

Comment:
There is a bug where this doesn't work unless you manually switch tabs first at least once